lib/firmata.js

```diff
@@ -505,6 +523,12 @@ function Board(port, options, callback) {
     RES_TX3: 0x07,
   };
 
+  this.RESOLUTION = {
+    ADC: null,
```

Comment:
I guess another way to think about this is do the default values live in firmata.js or in johnny-five (where the normalization of analog and pwm values takes place).